compiler/rustc_parse/src/parser/mod.rs

```diff
@@ -18,8 +18,10 @@ use rustc_ast::ptr::P;
 use rustc_ast::token::{self, DelimToken, Token, TokenKind};
 use rustc_ast::tokenstream::{self, DelimSpan, TokenStream, TokenTree, TreeAndSpacing};
 use rustc_ast::DUMMY_NODE_ID;
-use rustc_ast::{self as ast, AttrStyle, AttrVec, Const, CrateSugar, Extern, Unsafe};
-use rustc_ast::{Async, MacArgs, MacDelimiter, Mutability, StrLit, Visibility, VisibilityKind};
+use rustc_ast::{self as ast, AnonConst, AttrStyle, AttrVec, Const, CrateSugar, Extern, Unsafe};
+use rustc_ast::{
+    Async, Expr, ExprKind, MacArgs, MacDelimiter, Mutability, StrLit, Visibility, VisibilityKind,
+};
```

Comment:
I think @petrochenkov wants

```suggestion
use rustc_ast::{Async, Expr, ExprKind, MacArgs, MacDelimiter, Mutability, StrLit};
use rustc_ast::{Visibility, VisibilityKind};
```

which matches the style in this file. If we want imports to be uniformly formatted across the codebase, we should adjust our `rustfmt.toml` accordingly and do that in one big PR